source/software/matlab_parallel_computing.rst

```diff
@@ -7,119 +7,99 @@ General
-the previous step again. All of these properties are available in the ``AdditionalProperties`` property of your cluster profile. 
-   
-Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+You can then view and edit the additional properties:
 
 ::
 
-   >> c = parcluster;
-   
-``c = parcluster`` will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
-the correct profile will be chosen when using this command.
+    $ # Access
+    $ c.AdditionalProperties
+    $
+    $ #edit
+    $ c.AdditionalProperties.<property_name> = '<property_value>'
 
-If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+One of these properties you will always have to set, namely `AccountName`. Set this to the name of your credit account.
+
+Next to this, it is also possible to change the more standard parameters of the cluster profile. One of these is the `JobStorageLocation` parameter. This defaults
+to a location in your `$VSC_HOME`. It is however strongly recommended to change this to another location, e.g. `$VSC_DATA`. Changing this, or any of
+the other cluster profile parameters can be done as follows:
 
 ::
 
-   >> c = parcluster('local');
+    $ # Access
+    $ c.JobStorageLocation = '</path/to/your/custom/jobdir>'
 
-You can set a range of additional properties on each profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
-other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+Both after changing any of the additional properties or the standard parameters, you will have to save that profile to keep the changes after closing your
+session:
 
 ::
 
-   >> % View the additional properties
-   >> c.AdditionalProperties
-   >> % set the required properties
-   >> c.AdditionalProperties.AccountName = <account-name>;
-   >> c.AdditionalProperties.WallTime = '05:00:00';
-   >> % save the profile
-   >> c.saveProfile
-   
-Now the profile is ready, and you should be able to submit multi-node jobs with Matlab now.
+    $ c.saveProfile;
+
 
 Submitting jobs
 +++++++++++++++
 
 Submitting jobs will happen from within Matlab. There are two ways to do this. You can start a Matlab session on a login node, or you can first request an
 interactive job, where you launch your Matlab session. Which one you choose will depend on the workload you plan to have on the node before submitting a job.
-As for any other work on the cluster, you can always do minor work on the login node, but for heavier calculations (before submitting your multinode job), you should
-use an interactive job. 
+As for any other work on the cluster, you can always do minor work on the login node, but for heavier calculations (before submitting your effective job), you should
```

Comment:
this sentence contradicts the first sentence of the paragraph.
What shall the users eventually do? Can they use the login node for this or not?
If not, the first sentence needs a polish too.